packages/gossiplog/src/AbstractGossipLog.ts

```diff
@@ -189,7 +189,7 @@ export abstract class AbstractGossipLog<Payload = unknown> extends TypedEventEmi
 		const { reverse = false, limit, ...where } = range
 		return this.db.query<{ id: string; signature: Signature; message: Message<Payload> }>("$messages", {
 			where: { id: where },
-			select: { id: true, signature: true, message: true },
```

Comment:
Should we add branch to the return type above?

Reply:
Yes, good catch